### PR TITLE
"clear" text added underneath the "Previous Search". The "clear" will clear search history if clicked.

### PR DESCRIPTION
**Pull request by Meechlouch**
I added a clear history button. I added margin to css file to seperate the two button, also added an id on the html page.

### CODE REVIEW

File: assets/script/script.js [hunk 1]
```diff
@@ -31,6 +31,15 @@ $(document).ready(function () {
       }
     }
   });
+
+  //Listener on clear button clears the history list
+  $(".shiftRight").on("click", function () {
+    window.localStorage.clear("history");
+    window.sessionStorage.clear();
```
Comment (nobleburgundy) [inline assets/script/script.js]: I think we should update the html for this to be a link with an id. That way we can listen here for something like...
```
$("#clearLink").on("click", ...
```
.shiftRight is a class and could be used elsewhere.

File: index.html [hunk 2]
```diff
@@ -43,6 +44,9 @@ <h1 class="nameTemp has-text-warning is-size-1 is-size-4-mobile"></h1>
                   <div id="history">
                     <br>
                     <h2 class="has-text-warning is-size-3 ">Previous Searches</h2>
+                    <strong class="shiftRight">
+                      <h4><em><b>CLEAR</b></em></h4>
+                    </strong>
```
Comment (nobleburgundy) [inline index.html]: Can we update this to a link perhaps, so we get the hover over cursor? Also, let's add an id to that 'link' - or button if youd rather go with a button, but I think a link here works. Something like this...
```
<a class="shiftRight" id="clearLink"><em>CLEAR</em></a>
```
This way you can remove the <strong> tag as well and just place this under the Previous Searches h2.

File: assets/script/script.js (hunk 1 above)
Comment (nobleburgundy) [inline assets/script/script.js:38]: Also, are we using sessionStorage anywhere? I think we're just using localStorage to save the history right? So, we dont want to clear the sessionStorage too if we're not using it.

**Review by nobleburgundy — CHANGES_REQUESTED** [review]

File: assets/script/script.js (hunk 1 above)
Comment (Meechlouch) [reply assets/script/script.js:38]: No, the sessionStorage is not being used, i forgot to erase it.

Comment (Meechlouch) [reply assets/script/script.js]: Yes, I agree.

File: index.html (hunk 2 above)
Comment (Meechlouch) [reply index.html]: I would need assistance with the hover as i never used that code b4. Maybe i'll pass this to Mariam.

Comment (nobleburgundy) [reply index.html]: Let me know if you want to walk through it together - we can pair program it out.

**Review by nobleburgundy — APPROVED** [review]

**Pull request merged**